src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.cs

```diff
@@ -117,9 +128,6 @@ internal void TearDown()
 
         _isSetUp = false;
         GC.SuppressFinalize(this);
-
-        this.Uninitialize();
-        this.RemoveServices();
```

Comment:
no need for these.  all they did was null out managed objects. something unnecessary with managed types and disposal.